rdoc/gosu.rb

```diff
@@ -390,6 +390,18 @@ def self.load_tiles(source, tile_width, tile_height, options = {}); end
     # @return [Image?] an image that represents a portion of the containing image
     def subimage(left, top, width, height); end
 
+    ##
+    # Returns an image from a binary string of packed RGBA values. (e.g. from (Image#to_blob))
+    #
+    # @param blob [String] a binary string with 1 byte per channel and pixel (RGBA) (width * height * 4)
```

Comment:
I think I might have added that already - but the old image constructor also supported 4 byte per channel for floating point values, `from_blob` should als officially support it as it doesn't cost us much